node/silkworm/downloader/internals/header_persistence.cpp

```diff
@@ -203,7 +205,7 @@ void HeaderPersistence::update_canonical_chain(BlockNum height, Hash hash) {  //
         ancestor_hash = ancestor->parent_hash;
         ancestor_height--;
```

Comment:
--ancestor_height; instead of ancestor_height--; (google style)

Reply:
done